pipeline/profile-controller/base/configmap.yaml

```diff
@@ -22,26 +27,74 @@ data:
                     "apiVersion": "v1",
                     "kind": "Secret",
                     "metadata": {
-                    "name": "mlpipeline-minio-artifact",
-                    "namespace": namespace
+                        "name": "mlpipeline-minio-artifact",
+                        "namespace": namespace,
                     },
                     "data": {
                         "accesskey": "bWluaW8=", # base64 for minio
-                        "secretkey": "bWluaW8xMjM=" # base64 for minio123
-                    }
+                        "secretkey": "bWluaW8xMjM=", # base64 for minio123
+                    },
                 },
                 {
                     "apiVersion": "v1",
                     "kind": "ConfigMap",
                     "metadata": {
-                    "name": "metadata-grpc-configmap",
-                    "namespace": namespace
+                        "name": "metadata-grpc-configmap",
+                        "namespace": namespace,
                     },
                     "data": {
-                    "METADATA_GRPC_SERVICE_HOST": "metadata-grpc-service.kubeflow",
-                    "METADATA_GRPC_SERVICE_PORT": "8080"
-                    }
-                }
+                        "METADATA_GRPC_SERVICE_HOST": "metadata-grpc-service.kubeflow",
+                        "METADATA_GRPC_SERVICE_PORT": "8080",
+                    },
+                },
+                {
+                    "apiVersion": "apps/v1",
+                    "kind": "Deployment",
+                    "metadata": {
+                        "labels": {"app": "ml-pipeline-visualizationserver"},
+                        "name": "ml-pipeline-visualizationserver",
+                        "namespace": namespace,
+                    },
+                    "spec": {
+                        "selector": {
+                            "matchLabels": {"app": "ml-pipeline-visualizationserver"},
+                        },
+                        "template": {
+                            "metadata": {
+                                "labels": {"app": "ml-pipeline-visualizationserver"},
+                                "annotations": {"sidecar.istio.io/inject": "false"},
```

Comment:
TODO: we need to turn on these to avoid cross-namespace access (visualization server allows running arbitrary code)
It's fine just adding a comment right now, we can first get it working e2e

Reply:
Right, that's a good point!